### DISCUSSION TITLE
Replicate results of tts_models/en/vctk/vits

### DISCUSSION THREAD

**Discussion by VSydorskyy** [coqui-ai/TTS#2673] [General Q&A]
I have downloaded `tts_models/en/vctk/vits` from pretrained model zoo and it sounds pretty descent 

Afterwards I have tried to train it by myself using [next config](https://github.com/coqui-ai/TTS/blob/dev/recipes/vctk/vits/train_vits.py) but results are much worse

After comparing generated `config.json` from my train and from pre-trained model I have found out several differences. Although, it seems that pretrained model was trained using 2 stage training. 

Maybe you can advice how can I replicate original results from `tts_models/en/vctk/vits` ? 

Thanks in advance!

**Comment by francois-vz** [comment]
I am having a very similar issue, the pretrained `tts_models/en/ljspeech/tacotron2-DDC_ph` sounds very good, but when I try to retrain it using the full LJSpeech corpus and the script `TTS/recipes/ljspeech/tacotron2-DDC/train_tacotron_ddc.py` my model is sounding much worse than the default. A bit more clarity from the devs side would be great.

```python
audio_config = BaseAudioConfig(
    sample_rate=22050,
    do_trim_silence=True,
    trim_db=60.0,
    signal_norm=False,
    mel_fmin=0.0,
    mel_fmax=8000,
    spec_gain=1.0,
    log_func="np.log",
    ref_level_db=20,
    preemphasis=0.0,
)

config = Tacotron2Config(  # This is the config that is saved for the future use
    audio=audio_config,
    batch_size=64,
    eval_batch_size=16,
    num_loader_workers=4,
    num_eval_loader_workers=4,
    run_eval=True,
    test_delay_epochs=-1,
    r=6,
    gradual_training=[[0, 6, 64], [10000, 4, 32], [50000, 3, 32], [100000, 2, 32]],
    double_decoder_consistency=True,
    epochs=1000,
    text_cleaner="phoneme_cleaners",
    use_phonemes=True,
    phoneme_language="en-us",
    phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
    precompute_num_workers=8,
    print_step=25,
    print_eval=True,
    mixed_precision=False,
    output_path=output_path,
    datasets=[dataset_config],
)
```
